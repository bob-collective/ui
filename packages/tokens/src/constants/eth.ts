import { ChainId } from '@gobob/chains';

import { USDC, USDT, WBTC } from './common';
import { WETH9 } from './weth';

export const ethereumTokens = {
  weth: WETH9[ChainId.ETHEREUM],
  usdt: USDT[ChainId.ETHEREUM],
  usdc: USDC[ChainId.ETHEREUM],
  wbtc: WBTC[ChainId.ETHEREUM]
};
