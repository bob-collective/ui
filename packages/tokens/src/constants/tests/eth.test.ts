import { describe, it, expect } from 'vitest';

import { ethereumTokens } from '../eth';

describe('Ethereum Tokens Snapshot', () => {
  it('should match snapshot for ethereumTokens', () => {
    expect(ethereumTokens).toMatchSnapshot();
  });
});
