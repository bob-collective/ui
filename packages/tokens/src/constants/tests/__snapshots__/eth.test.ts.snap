// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Ethereum Tokens Snapshot > should match snapshot for ethereumTokens 1`] = `
{
  "usdc": ERC20Token {
    "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
    "chainId": 1,
    "decimals": 6,
    "isNative": false,
    "isToken": true,
    "name": "USD Coin",
    "projectLink": undefined,
    "symbol": "USDC",
  },
  "usdt": ERC20Token {
    "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
    "chainId": 1,
    "decimals": 6,
    "isNative": false,
    "isToken": true,
    "name": "Tether USD",
    "projectLink": undefined,
    "symbol": "USDT",
  },
  "wbtc": ERC20Token {
    "address": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
    "chainId": 1,
    "decimals": 8,
    "isNative": false,
    "isToken": true,
    "name": "Wrapped BTC",
    "projectLink": undefined,
    "symbol": "WBTC",
  },
  "weth": ERC20Token {
    "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
    "chainId": 1,
    "decimals": 18,
    "isNative": false,
    "isToken": true,
    "name": "Wrapped Ether",
    "projectLink": "https://weth.io",
    "symbol": "WETH",
  },
}
`;
